splinterd/src/config/scabbard_storage.rs

```diff
@@ -14,9 +14,13 @@
 
 use std::{fmt::Display, str::FromStr};
 
-#[derive(Debug, Copy, Clone)]
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug, Copy, Clone)]
 pub enum ScabbardStorage {
+    #[serde(rename = "database")]
```

Comment:
Should this be a TOML-specific struct, like the logging config?

Reply:
Almost certainly, Ill make the change